src/clientManager.js

```diff
@@ -1,12 +1,13 @@
 
-const initializeClient = async ({address = harnessConfigs.address, port = harnessConfigs.port, log, setupDB, uuid = harnessConfigs.clientUuid, pem = harnessConfigs.clientPem} = {}) => {
+const initializeClient = async ({address = harnessConfigs.address, port = harnessConfigs.port, log, setupDB, pem = harnessConfigs.masterPrivateKey} = {}) => {
 
-    const api = bluzelle({
+    const api = await bluzelle({
         entry: `ws://${address}:${port}`,
-        uuid: uuid,
+        ethereum_rpc: harnessConfigs.ethereumRpc,
+        contract_address: harnessConfigs.esrContractAddress,
         private_pem: pem,
-        log: log,
-        p2p_latency_bound: 100
+        _connect_to_all: true,
+        log: false,
```

Comment:
Was this intentional, even though we have a `log` parameter still?

Reply:
Not intentional, as the other issue above, I did some clean up and didn't attempt a run. Sorry!! Fixed already on my local.